packages/components/src/drop-zone/provider.js

```diff
@@ -38,9 +39,9 @@ function getDragEventType( { dataTransfer } ) {
 
 function isTypeSupportedByDropZone( type, dropZone ) {
 	return (
-		( type === 'file' && dropZone.onFilesDrop ) ||
-		( type === 'html' && dropZone.onHTMLDrop ) ||
-		( type === 'default' && dropZone.onDrop )
+		( type === 'file' && !! dropZone.onFilesDrop ) ||
+		( type === 'html' && !! dropZone.onHTMLDrop ) ||
+		( type === 'default' && !! dropZone.onDrop )
```

Comment:
Minor observation: perhaps it's more straightforward to just wrap the whole return expression in a `Boolean( )` or prefix it with `!!` rather than prefix all the individual terms.

Reply:
Sure, that works too